airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshTunnel.java

```diff
@@ -310,7 +310,7 @@ KeyPair getPrivateKeyPair() throws IOException, GeneralSecurityException {
     if (keyPairs != null && keyPairs.iterator().hasNext()) {
       return keyPairs.iterator().next();
     }
-    return null;
+    throw new GeneralSecurityException("Unable to load private key pairs, please verify key pairs are properly inputted");
```

Comment:
Just making sure that there's somebody catching this exception up the stream

Reply:
Yes, this is caught within the [openTunnel](https://github.com/airbytehq/airbyte/blob/035f353b438475319236b85fbba56456548031f9/airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshTunnel.java#L337-L366) method